CHANGELOG.md

```diff
@@ -122,6 +122,9 @@ Main (unreleased)
 
 - Fixed a bug where converting `YACE` cloudwatch config to river skipped converting static jobs. (@berler)
 
+- Fixed the `agent_prometheus_scrape_targets_gauge` incorrectly reporting all discovered targets
```

Comment:
I think the bug wasn't included in the previous release (right?), so we may not need the entry here.

Reply:
The old bug is present in all previous versions (but only affecting users when they have clustering on). 
The new bug I've added in https://github.com/grafana/agent/pull/5263 is present in `0.36.2` unfortunately.